### ISSUE TITLE
The build process should minify css and js files

### ISSUE THREAD

**Issue by R-J** [vanilla/vanilla-cli#121]
The build process should optimize the css and js files. After building the theme boilerplate, you can see that there are multiple repeating doc blocks in the css.

Since the build process should be the last step before the theme is used in production, there is no need for "readable" and nicely formatted css and js assets. Instead they should be compresses to optimize load time.

@charrondev advised to create an issue and already gave a hint to a [solution for minifying the css file](https://github.com/vanilla/docs/pull/315#issuecomment-428178523)

**Comment by charrondev** [comment]
I've actually looked into this a little bit further.

## Minification

The built JS and CSS files are already minified. The issue you seem to be encountering is that we have too many license declarations coming into the built CSS files.

Our intention is to have 1 license remaining per individual project involved. It goes against most project licenses to remove their license declaration which is why the minifiers/uglifiers preserve them.

**TL;DR** My understanding is that we do NOT want to strip out license/copyright declarations which is the only comment left in. The code is otherwise unminified. We can look into lowering the number of time these licenses are repeated in our scss source, but we 100% cannot strip all license declarations.
